### PR TITLE
Add `generateServerUtils` code guard

### PR DESCRIPTION
**Pull request by pahaz**
As you can see we have some dead code.

---
**Q**: Why do you need to add a code guard? 
**A**: Look inside file changes

### CODE REVIEW

File: apps/condo/domains/billing/utils/serverSchema/index.js [hunk 1]
```diff
@@ -97,9 +90,7 @@ module.exports = {
     BillingAccountMeter,
     BillingAccountMeterReading,
     BillingReceipt,
-    BillingOrganization,
     ResidentBillingReceipt,
-    BillingCurrency,
```
Comment (pahaz) [inline apps/condo/domains/billing/utils/serverSchema/index.js:102]: `undefined`!

File: packages/codegen/generate.server.utils.js [hunk 2]
```diff
@@ -142,6 +142,8 @@ async function execGqlWithoutAccess (context, { query, variables, errorMessage =
 }
 
 function generateServerUtils (gql) {
+    if (!gql) throw new Error('you are trying to generateServerUtils without gql argument')
```
Comment (pahaz) [inline packages/codegen/generate.server.utils.js:145]: the guard!

Comment (Alllex202) [reply packages/codegen/generate.server.utils.js:145]: Probably the same check is needed in client and test utilities

Comment (pahaz) [reply packages/codegen/generate.server.utils.js:145]: In another PR

File: apps/condo/domains/billing/utils/serverSchema/index.js [hunk 3]
```diff
@@ -11,7 +11,6 @@ const { execGqlWithoutAccess } = require('@open-condo/codegen/generate.server.ut
 const { find } = require('@open-condo/keystone/schema')
 
 const { PAYMENT_DONE_STATUS, PAYMENT_WITHDRAWN_STATUS } = require('@condo/domains/acquiring/constants/payment')
-const { Payment } = require('@condo/domains/acquiring/gql')
```
Comment (toplenboren) [inline apps/condo/domains/billing/utils/serverSchema/index.js:14]: Why delete payment gql? Am I missing something?

Comment (pahaz) [reply apps/condo/domains/billing/utils/serverSchema/index.js:14]: No any usage

**Review by dkoviazin — APPROVED** [review]

**Review by toplenboren — APPROVED** [review]

**Pull request merged**